pkg/traceability/muleemitter.go

```diff
@@ -40,18 +46,22 @@ type MuleEventEmitterJob struct {
 }
 
 // NewMuleEventEmitter - Creates a client to poll for events.
-func NewMuleEventEmitter(eventChannel chan string, client anypoint.AnalyticsClient) *MuleEventEmitter {
-	return &MuleEventEmitter{
+func NewMuleEventEmitter(mulesoftConfig *config.MulesoftConfig, eventChannel chan string, client anypoint.AnalyticsClient) *MuleEventEmitter {
```

Comment:
Since the only value being used off the config object is the CachePath, let's pass just that value in and not the whole object.